frontend/packages/jobs/package.json

```diff
@@ -28,6 +30,7 @@
     "dev": "tsc --watch",
     "fmt": "pnpm run '/^fmt:.*/'",
     "fmt:biome": "biome check --write --unsafe .",
+    "generate-schema": "json-schema-to-zod --input ./src/prompts/JSON_SCHEMA.json --output ./src/prompts/JSON_SCHEMA.generated.ts",
```

Comment:
oh sorry , I'll remove this

Reply:
https://github.com/liam-hq/liam/pull/1058/commits/c85f9de9afdd85a152c53557caa8c9296bb53364